test/src/test/java/org/corfudb/infrastructure/logreplication/LogReplicationFSMTest.java

```diff
@@ -72,6 +72,7 @@ public class LogReplicationFSMTest extends AbstractViewTest implements Observer
     private static final int CORFU_PORT = 9000;
     private static final int TEST_TOPOLOGY_CONFIG_ID = 1;
     private static final String TEST_LOCAL_CLUSTER_ID = "local_cluster";
+    private static final int MAX_SNAPSHOT_ENTRIES_APPLIED = 50;
```

Comment:
can we have a public static constant defined to be used here as well as the ServerContext area?